src/mobile/components/marketing_signup_modal/index.jade

```diff
@@ -3,7 +3,7 @@ section.marketing-signup-modal
     img.marketing-signup-modal-img( src=modal.image )
     h1.marketing-signup-modal-copy= modal.copy
     a.avant-garde-box-button.avant-garde-box-button-gray.marketing-signup-modal-signup(
-      href='/sign_up?email=1&acquisition_initiative=' + modal.slug
+      href='/sign_up?acquisition_initiative=' + modal.slug
```

Comment:
Since the GDPR checkboxes are on the combined page now, we should redirect them there instead of the email page directly:

![image](https://user-images.githubusercontent.com/2236794/40502817-280bb0c8-5f5a-11e8-9f52-b40130c13854.png)